### PR TITLE
FEAT: Commons IP schema validation

### PR DESCRIPTION
**Pull request by carlwilson**
- removed expansions of status enums; and
- added a `ruleId` alias for validation `rule_id`.

### CODE REVIEW

**Review by dockmd — APPROVED** [review]
LGTM, however why did you remove spaces from well and not well formed constants?

**Comment by carlwilson** [comment]
> LGTM, however why did you remove spaces from well and not well formed constants?

That's because they're incompatible with commons-ip which doesn't include spaces. I need compatibility to auto compare the results via a JSON deserialisation. Once we have compatibility we can talk about changes, but I can't change commons-ip.

**Comment by carlwilson** [comment]
Final time I promise @dockmd, this is now compatible with commons-ip JSON and the new validator rest service works.

File: eark_validator/structure.py [hunk 1]
```diff
@@ -40,6 +39,7 @@
 
 METS_NAME = 'METS.xml'
 STR_REQ_PREFIX = 'CSIPSTR'
+ROOT = 'root '
```
Comment (dockmd) [inline eark_validator/structure.py:42]: Perhaps it is better to use ROOT without the space at the end and then combine it with f‘{ROOT} {self.name}’

**Review by dockmd — APPROVED** [review]

**Comment by carlwilson** [comment]
@dockmd, I should have kept my mouth shut. Found another change to make to coerce a final value. I'll take a look at your suggestion  about root and get a final commit up shortly. Looking to close this and merge to integrations ASAP as I can then get a test package out and an updated REST validator.

**Review by dockmd — APPROVED** [review]

**Pull request merged**